### PR TITLE
✔ Anthropic Model added

### PR DESCRIPTION
**Pull request by Nil2000**
According to Issue [#60](https://github.com/miurla/morphic/issues/60), I have added the Anthropic model.
Please note before running the application make sure to have these env variables
- ANTHROPIC_API_BASE
- ANTHROPIC_API_KEY
- ANTHROPIC_API_MODEL_ID

### CODE REVIEW

**Review by miurla — CHANGES_REQUESTED** [review]
Thank you for the PR.

Instead of commenting out the switch of the provider, let's make it possible to branch with a config.

.env.local
`SPECIFIC_PROVIDER={'anthropic'}`

writer.tsx
```
if (prosess.env.SPECIFIC_PROVIDER === 'anthropic' ){
 // anthropic
} else {
 // openai
}
```


Please remove ANTHROPIC_API_XX and use SPECIFIC_API_XX instead.

This PR will also require changes to the README and .env.example

File: package-lock.json
Comment (miurla) [inline package-lock.json:1]: package-lock.json isn't needed.
Instead of npm, please use bun to install `@ai-sdk/anthropic`.

**Comment by Nil2000** [comment]
Ok as per your requirements, I changed the `writer.ts` file.
For `Readme.md` and `.env.local.example`, I also added the `SPECIFIC_PROVIDER`.
Is this fine?

**Comment by miurla** [comment]
> Ok as per your requirements, I changed the `writer.ts` file. For `Readme.md` and `.env.local.example`, I also added the `SPECIFIC_PROVIDER`. Is this fine?

It seems that it hasn't been committed.

https://github.com/miurla/morphic/pull/79/commits

**Comment by miurla** [comment]
The changes are OK.
Since bun.lock is conflicting, could you commit after running the following command?


```
rm bun.lockb
bun i
```

**Comment by Nil2000** [comment]
Yeah sure

**Review by miurla — APPROVED** [review]

**Comment by miurla** [comment]
@Nil2000 

It seems there is still a conflict, so please merge with the latest `miurla:main` branch.


If there is still a conflict, could you please check that the version of bun is v1.1.4, resolve it again with the following command, and commit?

```
   rm bun.lockb
   bun install
```

**Pull request closed without merging**